packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -205,17 +213,61 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
    * @param commitData - Commit to be verified
    * @param context - Ceramic context
    * @param controller - DID value
+   * @param streamId - Stream ID for the commit
    * @private
    */
   async _verifySignature(
     commitData: CommitData,
     context: Context,
-    controller: string
+    controller: string,
+    streamId: StreamID
   ): Promise<void> {
+    const cacao = await this._verifyCapability(commitData, context, streamId)
+
     await context.did.verifyJWS(commitData.envelope, {
       atTime: commitData.timestamp,
       issuer: controller,
       disableTimecheck: commitData.disableTimecheck,
+      capability: cacao,
     })
   }
+
+  /**
+   * Verifies capability attached to a signed commit
+   * @param commitData - Commit to be verified
+   * @param context - Ceramic context
+   * @param streamId - Stream ID for the commit
+   * @returns true if capability was present and was verified, false otherwise
+   */
+  async _verifyCapability(
+    commitData: CommitData,
+    context: Context,
+    streamId: StreamID
+  ): Promise<Cacao | undefined> {
```

Comment:
nitpick, but might be better the return `null` instead of `undefined` if there's no capability?